app/logger/logger.go

```diff
@@ -0,0 +1,43 @@
+package logger
+
+import (
+	"log"
+
+	"go.uber.org/zap"
+)
+
+type Wrapper struct{ log *zap.SugaredLogger }
+
+func NewLogger() *Wrapper {
```

Comment:
This func is inside logger package. 
It will be a bit more clear if we call this func just New(). 
In such a case, in a place where we are making a call to it looks like this: `logger.New()`

Reply:
Cool. I was walking around this decision and did not dare to do this